### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by vramperez**

### CODE REVIEW

File: db.js [hunk 1]
```diff
@@ -77,17 +77,19 @@ exports.init = function (callback) {
...
  */
 exports.addToken = function (token, callback) {
+    var error = 'Error adding the acces token "' + token + '" .';
+
     db.run('DELETE FROM token', function (err) {
         if (err) {
-            return callback('Error adding the acces token "' + token + '" .');
+            return callback(error);
         } else {
             db.run('INSERT OR REPLACE INTO token \
```
Comment (fdelavega) [inline db.js]: You are deleting the entry, so the replace has no sense

File: test/integration/accounting_CB_spec.js [hunk 2]
```diff
@@ -230,19 +230,19 @@ after(function (done) {
...
+                });
             });
-        });
 
-        after(function () {
-            task_callback();
-        });
+            after(function () {
+                task_callback();
+            });
```
Comment (fdelavega) [inline test/integration/accounting_CB_spec.js]: Mixed tabs and spaces

File: acc_modules/call.js [hunk 3]
```diff
@@ -65,8 +65,8 @@ var count = function (countInfo, callback) {
     return callback(null, 1);
 };
 
-var getSpecification = function (callback) {
-    return callback(specification);
+var getSpecification = function () {
+    return specification;
```
Comment (fdelavega) [inline acc_modules/call.js:69]: This is nice but you need to update also the documentation

File: APIServer.js [hunk 4]
```diff
@@ -16,17 +16,19 @@ var crypto = require('crypto'),
  */
 exports.checkUrl = function (req, res) {
     req.setEncoding('utf-8');
+
     var bodyUrl = req.body.url;
+    var apiKey = req.get('X-API-KEY');
 
     if (!bodyUrl) {
         res.status(422).json({error: 'Url missing'});
```
Comment (fdelavega) [inline APIServer.js]: "Missing URL" is better

File: db_Redis.js [hunk 5]
```diff
@@ -289,19 +289,23 @@ exports.deleteAdmin = function (idAdmin, callback) {
             async.each(services, function (service, task_callback) {
```
Comment (fdelavega) [inline db_Redis.js:289]: For the future try to use camelcase for var names

File: test/integration/usageManagementAPI_spec.js [hunk 6]
```diff
@@ -221,343 +199,123 @@ describe('Testing the usage specification and usage notifications', function ()
...
+                    publicPath: publicPath,
+                    orderId: 'orderId1',
+                    productId: 'productId1',
+                    customer: 'user1',
+                    unit: unit,
+                    recordType: 'typeUsage'
+                }];
+                var accountings = [{
+                    apiKey: apiKey,
+                    value: 2
```
Comment (fdelavega) [inline test/integration/usageManagementAPI_spec.js:246]: Don't you have a all this constants already defined in a test data file?

File: test/integration/usageManagementAPI_spec.js [hunk 7]
```diff
@@ -221,343 +199,123 @@ describe('Testing the usage specification and usage notifications', function ()
...
+                    orderId: 'orderId2',
+                    productId: 'productId2',
+                    customer: 'user2',
+                    unit: unit,
+                    recordType: 'typeUsage'
+                }];
+                var accountings = [{
+                    apiKey: apiKey,
+                    value: 2
+                }];
```
Comment (fdelavega) [inline test/integration/usageManagementAPI_spec.js:285]: The same as before

File: test/unit/APIServer_spec.js [hunk 8]
```diff
@@ -84,17 +47,11 @@ describe('Testing APIServer', function() {
 
     describe('Function "getUnits"', function() {
 
-        it('should return 200 when the response contains the accounting units supported', function() {
+        it('should return 200 when the response contains the accounting units supported', function(done) {
+
             var modules = ['cal', 'megabyte'];
```
Comment (fdelavega) [inline test/unit/APIServer_spec.js]: Not too much important for this test, but you have a typo "call"

**Comment by vramperez** [comment]
New commits added. I will refactor the rest of the tests after fixing the issues.

**Pull request merged**